tests/php/ConfigTest.php

```diff
@@ -80,22 +80,27 @@ public function testGenerateTurnSettings() {
 
 		/** @var \PHPUnit_Framework_MockObject_MockObject|ISecureRandom $secureRandom */
 		$secureRandom = $this->createMock(ISecureRandom::class);
+		$secureRandom
+			->expects($this->once())
+			->method('generate')
+			->with(16)
+			->willReturn('abcdefghijklmnop');
 		$helper = new Config($config, $secureRandom, $timeFactory);
 
 		//
 		$server = $helper->getTurnSettings();
 		if ($server['server'] === 'turn.example.org') {
 			$this->assertSame([
 				'server' => 'turn.example.org',
-				'username' => '1479829425',
-				'password' => 'ZY8fZQxAw/24gT0XYnMlcepUFlI=',
+				'username' => '1479829425:abcdefghijklmnop',
+				'password' => 'eibuiLbHf+0lk9jnmP8fTXjgcCc=',
```

Comment:
1) OCA\Spreed\Tests\php\ConfigTest::testGenerateTurnSettings
Failed asserting that Array &0 (
    'server' => 'turn.example.org'
    'username' => '1479829425:abcdefghijklmnop'
    'password' => '4VJLVbihLzuxgMfDrm5C3zy8kLQ='
    'protocols' => 'udp,tcp'
) is identical to Array &0 (
    'server' => 'turn.example.org'
    'username' => '1479829425:abcdefghijklmnop'
    'password' => 'eibuiLbHf+0lk9jnmP8fTXjgcCc='
    'protocols' => 'udp,tcp'
).

/drone/src/github.com/nextcloud/server/apps/spreed/tests/php/ConfigTest.php:94

Reply:
Fixed, I was passing secret + data in the wrong order when creating the values for the testcase.